Tests/LeafKitTests/LeafTests.swift

```diff
@@ -295,76 +281,60 @@ final class LeafTests: XCTestCase {
 //
 //        let context = Stuff(title: "foo")
 //        let data = try TemplateDataEncoder().testEncode(context)
-//        try XCTAssertEqual(renderer.testRender(template, data), expected)
-//    }
-//
-//    func testInvalidForSyntax() throws {
-//        let data = try TemplateDataEncoder().testEncode(["names": ["foo"]])
-//        do {
-//            _ = try renderer.testRender("#for( name in names) {}", data)
-//            XCTFail("Whitespace not allowed here")
-//        } catch {
-//            XCTAssert("\(error)".contains("space not allowed"))
-//        }
-//
-//        do {
-//            _ = try renderer.testRender("#for(name in names ) {}", data)
-//            XCTFail("Whitespace not allowed here")
-//        } catch {
-//            XCTAssert("\(error)".contains("space not allowed"))
-//        }
-//
-//        do {
-//            _ = try renderer.testRender("#for( name in names ) {}", data)
-//            XCTFail("Whitespace not allowed here")
-//        } catch {
-//            XCTAssert("\(error)".contains("space not allowed"))
-//        }
-//
-//        do {
-//            _ = try renderer.testRender("#for(name in names) {}", data)
-//        } catch {
-//            XCTFail("\(error)")
-//        }
-//    }
-//
-//    func testTemplating() throws {
-//        let home = """
-//        #set("title", "Home")#set("body"){<p>#(foo)</p>}#embed("base")
-//        """
-//        let expected = """
-//        <title>Home</title>
-//        <body><p>bar</p></body>
-//
-//        """
-//        renderer.astCache = ASTCache()
-//        defer { renderer.astCache = nil }
-//        let data = try TemplateDataEncoder().testEncode(["foo": "bar"])
-//        try XCTAssertEqual(renderer.testRender(home, data), expected)
-//        try XCTAssertEqual(renderer.testRender(home, data), expected)
-//    }
-//
-//    // https://github.com/vapor/leaf/issues/96
-//    func testGH96() throws {
-//        let template = """
-//        #for(name in names) {
-//            #(name): index=#(index) last=#(isLast) first=#(isFirst)
-//        }
-//        """
-//        let expected = """
-//
-//            tanner: index=0 last=false first=true
-//
-//            ziz: index=1 last=false first=false
-//
-//            vapor: index=2 last=true first=false
-//
-//        """
-//        let data = try TemplateDataEncoder().testEncode([
-//            "names": ["tanner", "ziz", "vapor"]
-//            ])
-//        try XCTAssertEqual(renderer.testRender(template, data), expected)
+//        try XCTAssertEqual(render(template, data), expected)
 //    }
+
+    // TODO: WHY is whitespace not allowed here?!?
```

Comment:
This doesn't make any sense to me either haha. It should really be testing that whitespace _is_ allowed here.

Reply:
Ok - thought I was losing my mind when I read that test

Reply:
Addressing/adding in future PR concentrating on Tags